promptsource/templates/anli/templates.yaml

```diff
@@ -69,55 +78,178 @@ templates:
     - 'No'
     answer_choices_key: null
     id: 9b613182-c6ab-4427-9221-3d68f6d62765
-    jinja: '{{premise}} Based on the previous passage, is it true that {{hypothesis}}
+    jinja: '{{premise}} Based on the previous passage, is it true that "{{hypothesis}}"?
       Yes, no, or maybe? ||| {{ answer_choices[label] }}'
     metadata: !TemplateMetadata
       _do_eval: true
       _do_train: false
       choices_in_prompt: true
-      metrics: []
+      metrics:
+      - Accuracy
       original_task: true
     name: based on the previous passage
     reference: "Adapted from the BoolQ prompts in Schick & Sch\xFCtze 2021."
-  cf55f09b-bc68-439d-a9b2-781263509f99: !Template
+  a850110d-f1a3-49b4-949a-d3bfe9f81344: !Template
     answer_choices:
-    - 'Yes'
-    - Neutral
-    - 'No'
+    - 'yes'
+    - maybe
+    - 'no'
     answer_choices_key: null
-    id: cf55f09b-bc68-439d-a9b2-781263509f99
-    jinja: 'Sentence 1: {{premise}}
-
-      Sentence 2: {{hypothesis}}
-
-      Question: Does Sentence 1 entail Sentence 2? Yes, No, or Neutral? |||
+    id: a850110d-f1a3-49b4-949a-d3bfe9f81344
+    jinja: '{{premise}} Are we justified in saying that "{{hypothesis}}"? Yes, no,
+      or maybe? ||| {{ ["yes", "maybe", "no"][label] }} '
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: justified in saying
+    reference: Webson & Pavlick 2021
+  bab86d5a-4f9c-40db-b619-a7b7d5cae681: !Template
+    answer_choices:
+    - 'true'
+    - inconclusive
+    - 'false'
+    answer_choices_key: null
+    id: bab86d5a-4f9c-40db-b619-a7b7d5cae681
+    jinja: 'Take the following as truth: {{premise}}
 
-      {{answer_choices[label]}}'
+      Then the following statement: "{{hypothesis}}" is {{"true"}}, {{"inconclusive"}},
+      or  {{"false"}}? ||| {{ answer_choices[label] }}'
     metadata: !TemplateMetadata
-      _do_eval: true
-      _do_train: false
       choices_in_prompt: true
       metrics: []
       original_task: true
-    name: does S1 entail S2?
-    reference: Copied from Victor's prompts for XNLI.
-  e0e8b914-c32c-4c85-8b8f-0f61ae79e2b3: !Template
+    name: take the following as truth
+    reference: Bers et al.
+  bcd90047-3a2b-426b-b065-8a418f1317b8: !Template
     answer_choices:
-    - must be true
-    - might be true
-    - must be false
+    - 'yes'
+    - maybe
+    - 'no'
     answer_choices_key: null
-    id: e0e8b914-c32c-4c85-8b8f-0f61ae79e2b3
-    jinja: Given that {{premise}}, it {{"must be true, might be true, or must be false"}}
-      that {{hypothesis}}? ||| It {{ answer_choices[label] }}.
+    id: bcd90047-3a2b-426b-b065-8a418f1317b8
+    jinja: 'Given that {{premise}} Therefore, it must be true that "{{hypothesis}}"?
```

Comment:
I think it would be slightly more grammatically correct to say "it therefore must be true..."

Reply:
I think they're equally correct? "It therefore…" sounds pedantic and Google Ngram says "therefore it" is twice as frequent. https://books.google.com/ngrams/graph?content=therefore+it%2Cit+therefore&year_start=1800&year_end=2019&corpus=26&smoothing=3#